docs/trestle_author.md

```diff
@@ -409,6 +409,48 @@ Running `trestle author docs validate -tn docs_task -gh="Governed section"` will
 - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
   Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
 
+### Validating the documents against different templates
+
+Validation against multiple templates as stated before can be done, but there is another scenario that you can leverage on trestle to have more than 1 instance of a template existing in the task folder.
+
+For that to happen you will need to provide your template with the following parameter at the yaml header level, matching the type of template to be implemented so the validation can occur:
+
+> x-trestle-template-type: insert_template_type_here
+
+Please, take into consideration that for the validation to happen you will need to provide the type in each template you want to implement as well as on each instance of the respective template. With that, you will be able to create more than 1 instance document per template and give the instance the name desired.
+
```

Comment:
the x-trestle-template-type header needs to be only added in the task (actual) documents. Do we need to put the header in template file also?

Reply:
done